docs/manual/lore/faction-proteron.md

```diff
@@ -0,0 +1,61 @@
+
+Some debate followed, but the consensus was that House Proteron had achieved its original goal, and now it was time for the weak Empire to make way for the new generation of galactic dominance. But when the Proteron High Autarch delivered this message to the Emperor, he was furious at the perceived insult and ordered all Autarchs executed. Appalled at this reaction, the Circle concluded in an emergency meeting that the Empire had no intention of honoring the original agreement, thereby forfeiting its claim to the loyalty of House Proteron. It was clear to the Circle that should the Empire insist on trying to turn the Proteron into another Dvaered incident or, worse still, turning them into Empire systems, there could be no peace with them.
+
+Knowing that they could not at the time resist the entire might of the Empire's infrastructure, the Proteron decided to plant their agents in every major Imperial agency while publicly grovelling at the Empire's feet. Soon enough, the Proteron were even in charge of the Empire's covert operations against the Proteron. Now having loosened their leash greatly, they set out to prepare for a galaxy-scale war. While they worked, the shape of the galaxy was changing, so to speak.
+
+New advances in quantum engineering led to the development of quantum transtators and translators, alternative travel engines capable of bypassing warp points and so changing conventional military doctrine, which recommended fortifying chokepoint systems on the warp network. Expecting correctly that the Empire had not even thought of this change, the Proteron decided to bypass their defences using the new hypergate being built in Sol. After all, once the Emperor and their successors were in their hands, the war would be essentially meaningless. They built a hypergate in secret in their home system, Protera. They seceded from Imperial authority as soon as the gate was built and moved a small fleet as a feint into the classical defences. When the Empire took the bait, the Sovereign Proteron Autarchy, as they now styled themselves, sent the most massive fleet ever seen through their hypergate.
+
+Then the Incident occured.
+
+After the dust settled, Protera was a mind-bending hellhole in space, Sol was unreachable, their planets, factories and fleets had been decimated and the Circle was mostly gone. Thankfully, enough of the hierarchy remained that society did not dissolve. The Proteron scientists set out to determine the cause of the Incident, and their best answer is that the large number of ships travelling all at once through the hypergate overloaded it, causing Sol to explode with a huge mass of material, causing a chain reaction that obliterated systems and changed the warp network. On the other hand, Protera was sucked through the hypergate, and the resultant tarnishing of the quantum fields in there and surrounding systems had turned it into wild space -- human and nature likewise twisted into things out of nightmares.
```

Comment:
The proteron don't actually know what's beyond the nebula, so they shouldn't know about wild space. They became completely isolated with the impenetrable wall, while their scientists should be able to figure out the cause, they should have lost touch with anything and just sort of sit there, with their scouts that head into the nebula never reporting back.

Reply:
1. Isn't wild space not after the nebula but rather to the side of it? Two routes through, one via Sol the other via Protera?
2. I thought wild space was a delayed effect (drove people mad, not made them mad instantly), so the Proteron could have observed their scouts behaving erratically on comms before losing contact.

Reply:
The way I planned it, wild space is "sandwiched" in the Nebula, and the route in is from soromid space, while the route out is at the entrance of proteron space. You sort of do a transition soromid -> nebula -> wild space -> nebula -> proteron/thurion. Furthermore, it uses hidden jumps in and out, so it's more like an isolated region that the player is almost the first to discover (although the POI will mention someone you got out).

Yeah, I guess the proteron should understand something bad is going on (and probably one of the reasons they don't move their main fleet through it), however, they probably should not know the extent and exact form of the problem.

Reply:
Alright, I'll make the claim about wild space implicit (no nebula but ships behave weird and vanish). All the same, wild space ought to be brought up somewhere. Currently the only faction with access is the SPA

Reply:
Strictly speaking, since the jumps are hidden, nobody has "access" to it, although it borders soromid/proteron. Maybe it's time for me to implement it (after I finish the Minerva campaign).

Reply:
Maybe we ought to keep a stellar_cartography doc with details of stuff like this, the haze etc.